docs/plugins.md

```diff
@@ -0,0 +1,262 @@
+Protractor Plugins
+=================
+
+Plugins extend Protractor's base features by using hooks during test
+execution to gather more data and potentially modify the test output.
+
+The Protractor API and available plugins are *BETA* and may change
+without a major version bump.
+
+The `plugins` folder contains default plugins for Protractor.
+
+##In this document:
+* Using Plugins
```

Comment:
Don't worry about that.  I have a [pending PR](https://github.com/angular/protractor/pull/1894) to deal with that.